src/styles.css

```
@@ -312,6 +312,23 @@ body { margin: 0; min-width: 320px; min-height: 100vh; overflow: hidden; }
+.github-integration-body { display: grid; gap: 12px; max-height: calc(100vh - 96px); padding: 14px; overflow: auto; background: var(--chrome); }
+.github-integration-card, .github-control-login { display: grid; gap: 10px; padding: 12px; border: 2px groove white; background: #f4f4f4; }
+.github-integration-card h3, .github-control-login h3, .github-integration-card p, .github-control-login p { margin: 0; }
+.github-control-login label, .github-integration-card label { display: grid; gap: 4px; font-weight: 700; }
+.github-control-login input, .github-integration-card select { min-width: 0; padding: 6px; font: inherit; }
+.github-status-row, .github-card-heading, .github-device-challenge { display: flex; align-items: center; justify-content: space-between; gap: 12px; }
+.github-status-row > span:first-child, .github-card-heading > span, .github-device-challenge { min-width: 0; }
+.github-status-row small, .github-card-heading small, .github-device-challenge small { display: block; margin-top: 3px; }
+.github-status-badge { padding: 3px 8px; border: 1px solid #1f6f3f; color: #174f30; background: #d9f5e4; font-weight: 700; }
+.github-install-link { justify-self: start; text-decoration: none; color: inherit; }
+.github-device-challenge { flex-wrap: wrap; padding: 10px; border: 1px solid #8b6d15; background: #fff7d6; }
+.github-device-challenge strong, .github-project-defaults dd { font-family: "SFMono-Regular", Consolas, "Liberation Mono", monospace; }
```

Stylelint (17.14.0)

[error] Expected no quotes around "SFMono-Regular" (font-family-name-quotes)

(font-family-name-quotes)